src/Dfe.PlanTech.Application/Submission/Queries/GetSubmissionStatusesQuery.cs

```diff
@@ -1,5 +1,6 @@
 ﻿using Dfe.PlanTech.Application.Persistence.Interfaces;
 using Dfe.PlanTech.Application.Submission.Interfaces;
```

Comment:
This looks like it might be an unused import:

`using Dfe.PlanTech.Application.Submission.Interfaces;`